src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -957,8 +957,10 @@ void WindowImplX11::setIcon(const Vector2u& size, const std::uint8_t* pixels)
 {
     // X11 wants BGRA pixels: swap red and blue channels
     // Note: this memory will be freed by X11Ptr<XImage> deleter
+    // NOLINTBEGIN(cppcoreguidelines-no-malloc)
     auto* iconPixels = static_cast<std::uint8_t*>(
         std::malloc(static_cast<std::size_t>(size.x) * static_cast<std::size_t>(size.y) * 4));
+    // NOLINTEND(cppcoreguidelines-no-malloc)
```

Comment:
This was pretty annoying. If you add an inline `// NOLINT` comment on the same line as the `std::malloc` call, clang-format ends up reformatting that line such that the `std::malloc` line is on the top line. If you switch to a `// NOLINTNEXTLINE` then clang-format keeps the current formatting where `std::malloc` is on the second line of the expression.